monai/transforms/compose.py

```diff
@@ -358,3 +358,124 @@ def inverse(self, data):
         for o in reversed(applied_order):
             data = apply_transform(self.transforms[o].inverse, data, self.map_items, self.unpack_items, self.log_stats)
         return data
+
+
+class SomeOf(Compose):
+    """
+    ``SomeOf`` provides the ability to randomly sample a fixed number or
+    up to a fixed number of transforms from a list of callables.
+
+    If sampling up to the max_num_transforms, each invocation of SomeOf uniformly samples a target number of transforms
+    for that invocation.
+
+    At each invocation the set of transforms to apply is uniformly sampled.
+
+    Args:
+        transforms: list of callables.
+        map_items: whether to apply transform to each item in the input `data` if `data` is a list or tuple.
+            Defaults to `True`.
+        unpack_items: whether to unpack input `data` with `*` as parameters for the callable function of transform.
+            Defaults to `False`.
+        log_stats: whether to log the detailed information of data and applied transform when error happened,
+            for NumPy array and PyTorch Tensor, log the data shape and value range,
+            for other metadata, log the values directly. Default to `False`.
+        min_num_transforms: minimum number of transforms to sample. Defaults to `0`.
+        max_num_transforms: maximum number of transforms to sample. Defaults to `3`.
+        fixed: whether to sample exactly `max_num_transforms` transforms, or up to it. Defaults to `False`.
+        replace: whether to sample with replacement. Defaults to `False`.
+        weights: weights to use in for sampling transforms. Will be normalized to 1. Default: None (uniform).
+    """
+
+    def __init__(
+        self,
+        transforms: Sequence[Callable] | Callable | None = None,
+        map_items: bool = True,
+        unpack_items: bool = False,
+        log_stats: bool = False,
+        min_num_transforms: int = 0,
+        max_num_transforms: int = 3,
```

Comment:
I'd suggest setting ```max_num_transforms``` to ```None```. When it is ```None```, it can internally set ```max_num_transforms``` to ```len(transforms)```